changelog/added-va108xx-support.md

```diff
@@ -0,0 +1,2 @@
+Added support for the Vorago VA108xx family of MCUs by adding a chip description file for the
+VA108xx family and a VA108XX specific ARM debug sequence
```

Comment:
There is not debug sequence impl in this PR 👀

Reply:
Classic copy & paste mistake, good spot :)

Reply:
Fixed